valohai_yaml/pipelines/conversion.py

```diff
@@ -7,29 +7,53 @@
     ExecutionNode,
     Node,
     Pipeline,
+    PipelineParameter,
     TaskNode,
 )
 from valohai_yaml.objs.pipelines.override import Override
 
 ConvertedObject = Dict[str, Any]
 
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+    class ConvertedPipeline(TypedDict):
+        """TypedDict for converted Pipeline object."""
+
+        edges: List[ConvertedObject]
+        nodes: List[ConvertedObject]
+        parameters: Dict[str, ConvertedObject]
+else:
+    ConvertedPipeline = ConvertedObject
+
+
 class PipelineConverter:
     """Converts pipeline objects to Valohai API payloads."""
 
     def __init__(
-        self,
-        *,
-        config: Config,
-        commit_identifier: str,
+            self,
+            *,
+            config: Config,
+            commit_identifier: str,
```

Comment:
I don't think change is necessary to get this feature out.

Would be nice to have a ruff rule to check the indention of function parameters.